### PR TITLE
[docs] A new connector property table is added to help users quickly …

### LINKED ISSUE #1735: [feature] A new connector property table is added to help users quickly understand the corresponding properties of different connectors
**Is your feature request related to a problem? Please describe.**
A new connector property table is added to help users quickly understand the corresponding properties of different connectors

**Describe the solution you'd like**
A new connector property table is added to help users quickly understand the corresponding properties of different connectors

**Describe alternatives you've considered**
A new connector property table is added to help users quickly understand the corresponding properties of different connectors

### PR DESCRIPTION
**Pull request by gtk96**
…understand the corresponding properties of different connectors

I don't know much about the characteristics of some connectors. Please help me to supplement them and point out the wrong information

this pr closed #1735

### CODE REVIEW

**Comment by rookiegao** [comment]
@leonardBang @Jiabao-Sun PTAL.

**Comment by leonardBang** [comment]
@gtk96 Sorry for lated reply, but the change looks good to me, would you like to open the PR and let me help to review ?

File: docs/content/about.md [hunk 1]
```diff
@@ -41,6 +41,20 @@ The following table shows the version mapping between Flink<sup>®</sup> CDC Con
 2. CDC connectors for DataStream API, users can consume changes on multiple databases and tables in a single job without Debezium and Kafka deployed.
 3. CDC connectors for Table/SQL API, users can use SQL DDL to create a CDC source to monitor changes on a single table.
 
+The following table shows the current features of the connector:
+
+| Connector                                    | No lock read | Concurrent read | exactly-once | Incremental snapshot |
```
Comment (leonardBang) [inline docs/content/about.md:46]:
```suggestion
| Connector                                    | Lock Database Table | Parallel Read | Exactly-once  Read| Incremental Snapshot Read |
```

File: docs/content/about.md [hunk 2]
```diff
@@ -41,6 +41,20 @@ The following table shows the version mapping between Flink<sup>®</sup> CDC Con
...
+|----------------------------------------------|--------------|-----------------|--------------|----------------------|
+| [mongodb-cdc](connectors/mongodb-cdc.md)     | true         | true            | true         | true                 |
```
Comment (leonardBang) [inline docs/content/about.md:48]: could we use ✅ and ❌  instead of `true` and `false` ?

**Review by leonardBang — COMMENTED** [review]
Thanks @gtk96 for the contribution! I left minor comments.

**Comment by gtk96** [comment]
@leonardBang done

**Review by leonardBang — APPROVED** [review]
Thanks @gtk96 for the timely update, LGTM

**Pull request merged**